flink/flink-1.14/src/main/java/io/tidb/bigdata/flink/connector/utils/JdbcUtils.java

```diff
@@ -59,7 +57,7 @@ public static JdbcConnectorOptions getJdbcOptions(Map<String, String> properties
         dbUrl.matches("jdbc:(mysql|tidb)://[^/]+:\\d+/.*"),
         "the format of database url does not match jdbc:(mysql|tidb)://host:port/.*");
     dbUrl = rewriteJdbcUrlPath(dbUrl, databaseName);
-    String driverName = dbUrl.startsWith(TIDB_URL_PREFIX) ? TIDB_DRIVER_NAME : MYSQL_DRIVER_NAME;
+    String driverName = TiDBDriver.driverForUrl(dbUrl);
```

Comment:
can you introduce what is the difference of jdbc:mysql and jdbc:tidb

Reply:
`jdbc:tidb` is used for TiDB-connector that you can find in TiBigData/jdbc. It is the exclusive connector for TiDB.
Because TiDB is compatible with MySQL and can use MySQL-connector, we can also use `jdbc:mysql`.
Btw, TiDB-connector is a wrapper of MySQL-connector in fact.